package.json

```diff
@@ -483,6 +489,11 @@
                     "when": "view == azureFunctionsExplorer && viewItem == applicationSettingItem",
                     "group": "1@3"
                 },
+                {
+                    "command": "azureFunctions.appSettings.toggleSlotSetting",
+                    "when": "view == azureFunctionsExplorer && viewItem == applicationSettingItem",
```

Comment:
I feel like we also want `config.azureFunctions.enableSlots == true` in this condition. What do you think?

Reply:
Yeah, that makes sense.  It'd be a pretty confusing setting if you didn't have slots enabled.  

That being said, I think the UI shows if a setting is a slot setting regardless of the `enableSlots` setting.  I'm not sure how they'd manage to do that-- maybe in the portal?